packs/datadog_agent/templates/pack.nomad.tpl

```diff
@@ -0,0 +1,146 @@
+[[- $ports := .my.ports ]]
+
+# see https://developer.hashicorp.com/nomad/docs/job-specification/job
+job "[[ .my.job_name ]]" {
+  region      = "[[ .my.region ]]"
```

Comment:
Should this support multi-region jobs (Enterprise feature)

Reply:
Good idea.

Tracking this in https://github.com/workloads/nomad-pack-registry/issues/3 for all Packs